src/modules/dashboard/DashboardTopPanel.tsx

```diff
@@ -110,7 +110,11 @@ export const DashboardTopPanel = () => {
         <TopInfoPanelItem icon={<NetAPYIcon />} title={<Trans>Net APY</Trans>} loading={loading}>
           {currentAccount ? (
             <FormattedNumber
-              value={(user?.earnedAPY || 0) - (user?.debtAPY || 0)}
+              value={
+                (user?.earnedAPY || 0) *
```

Comment:
Does it make sense to use big number here instead? Probably doesn't matter much but I fear there might be an overflow in certain Situations

Reply:
Since the string is already coming in as a floating point I don't think it makes sense to convert to integer -> BigNumber

I did add a divide by zero check that was missing though :sweat_smile:

Reply:
Yeah that would be way cleaner